src/asmcnc/apps/drywall_cutter_app/engine.py

```diff
@@ -737,9 +738,9 @@ def line_default_parameters(simulate=False):
             coordinates.append(coordinates[0])
 
             # Create a list of stepovers to add finishing passes
-            finish_passes = 0
-            finish_stepover = 0.5
-            finish_stepdown = self.config.active_config.cutting_depths.material_thickness
+            finish_passes = self.finishing_passes

```

Comment:
do you need additional variables? seems there would be 3 copies here, one in self, one here and one in circle_parameters

Reply:
I've used local vars along side self just so that if necessary they can be manipulated for a given context without editing the default values

Reply:
sure, I assumed so but couldn't see any usage. but assume it's possible it'll be used in future, so no probs.